packages/indexer-service/src/commands/start.ts

```diff
@@ -141,6 +142,12 @@ export default {
         required: false,
         group: 'Network Subgraph',
       })
+      .option('block-oracle-endpoint', {
```

Comment:
Does the indexer-service use the block oracle subgraph?

Reply:
I don't think so. This was added here because I wanted to make the subgraph endpoint a requirement for indexer management server. This requirement can be relaxed or use a default value?

Reply:
I would lean towards making the block oracle subgraph endpoint an optional config.

Reply:
For today I'm ok with setting a default on indexer-service and in subsequent PR we can remove requirement. :)

Reply:
ahhh just saw the second comment. I have pushed the updates to have optional epoch subgraph endpoint and removed from indexer-service and other unused instances :)

Reply:
Even better! Nice, I'll check it out now.